bootstrap/namespaces/helmfile/values/namespaces-sc.yaml.gotmpl

```diff
@@ -3,18 +3,15 @@ namespaces:
 - name: dex
 - name: opensearch-system
 - name: fluentd
-- name: influxdb-prometheus
```

Comment:
Won't this try to remove the namespace before the influxdb components have been uninstalled.
Have you checked so that there are no finalizes at play or leftover resources such as volumes?